11/umbraco-forms/editor/creating-a-form/form-settings.md

```diff
@@ -14,43 +14,65 @@ To access the Form Settings:
 
-    The following Validations are available:
-    
-    |Validation Type | Description|
-    |-|-|
-    | **Mandatory error message** | The error message is displayed for a field that is marked as mandatory but a value has not been provided upon submission. This setting can be overwritten on a field level - `{0}` will be replaced with the field caption. |
-    | **Invalid error message** | The error message is displayed for a field if the value provided is not valid (a regular expression has been setup but the input does not match). This setting can be overwritten on a field level - `{0}` will be replaced with the field caption. |
-    | **Show validation summary** | Enable this option if you wish to display a summary of all the error messages on top of the Form. |
-    | **Hide field validation labels** | Enable this option if you wish to hide individual field error messages from being displayed. |
-    |**Mark fields** | You can choose to not mark any fields or only mark mandatory or optional fields. |
-    | **Indicator** | Choose which indicator to use when a field has been marked as mandatory. The default indicator is `*` |
+The following Validations are available:
 
-- **Moderation**
+|Validation Type | Description|
+|-|-|
+| **Mandatory error message** | The error message is displayed for a field that is marked as mandatory but a value has not been provided upon submission. This setting can be overwritten on a field level - `{0}` will be replaced with the field caption. |
+| **Invalid error message** | The error message is displayed for a field if the value provided is not valid (a regular expression has been setup but the input does not match). This setting can be overwritten on a field level - `{0}` will be replaced with the field caption. |
+| **Show validation summary** | Enable this option if you wish to display a summary of all the error messages on top of the Form. |
+| **Hide field validation labels** | Enable this option if you wish to hide individual field error messages from being displayed. |
+|**Mark fields** | You can choose to not mark any fields or only mark mandatory or optional fields. |
+| **Indicator** | Choose which indicator to use when a field has been marked as mandatory. The default indicator is `*` |
 
-    Enabling this feature allows the moderator to approve or deny a Form before its published. Most Forms contain publicly shown entries such as blog post comments or submissions for a social campaign and you can decide what's being published online with the help of this option.
+### Automcomplete
 
-    ![Form settings validation](images/FormSettingsModeration.png)
+The automcomplete setting for the overall form can be changed from the default of "None" to "On" or "Off". Setting this explicitly will control how the browser offers automatic prompts to the user when completing the form.
+
+![Form settings validation](images/FormSettingsAutcomplete.png)
```

Comment:
The image is a little zoomed in compared to other images 🙈 Not an issue but mentioning it since the font looks quite big... 

![image](https://user-images.githubusercontent.com/82437098/222400974-ed867208-8722-4f82-a5a2-b743b78e381a.png)

Reply:
Not too sure what to do about this one, as the image itself is in the same proportion as the others.  Looks like maybe GitBook is zooming it up to fill the space, so maybe you have some options around that?

If not, could perhaps add some horizontal whitespace to it.

Reply:
Ah ok.. I'll look at GitBook image options once the PR is merged.. 💪